Source/WTF/wtf/CurrentTime.cpp

```diff
@@ -314,4 +340,40 @@ ApproximateTime ApproximateTime::now()
 #endif
 }
 
+ContinuousTime ContinuousTime::now()
+{
+#if OS(DARWIN)
+    return fromMachContinuousTime(mach_continuous_time());
+#elif OS(LINUX) || OS(OPENBSD)
+    struct timespec ts { };
+    clock_gettime(CLOCK_BOOTTIME, &ts);
+    return fromRawSeconds(static_cast<double>(ts.tv_sec) + ts.tv_nsec / 1.0e9);
+#else
+    static double lastTime = 0;
+    double currentTimeNow = currentTime();
+    if (currentTimeNow < lastTime)
+        return lastTime;
+    lastTime = currentTimeNow;
+    return fromRawSeconds(currentTimeNow);
```

Comment:
These logic is now reused for three times. We should extract this like `monotonicCurrentTime()` or something and use that to pass the result to `fromRawSeconds()`.